konduit-serving-common-tests/pom.xml

```diff
@@ -58,5 +58,38 @@
             <artifactId>jetty-server</artifactId>
             <version>9.4.19.v20190610</version>
         </dependency>
+        <dependency>
+            <groupId>org.reflections</groupId>
+            <artifactId>reflections</artifactId>
+            <version>0.9.9</version>
```

Comment:
Don't use hardcoded versions like this. Versions should go in the parent pom - there's already a version defined.
Use `<version>${reflections.version}</version>` instead